2024/COMMUNICATION.md

```diff
@@ -0,0 +1,27 @@
+# Communication
+
+## Discourse announcement
+
+Dear Nix Community,
+
+It is with excitement that we launch 2024's Nix Survey!
+
+https://survey.nixos.org/2024
+
+We are conducting this survey to improve our understanding of the community and what everyone cares about.
+We hope to use your responses to support Nix Teams in best serving the community.
```

Comment:
(Slight wording) - We plan to use your responses to support Nix Teams to better serve the community.

Reply:
```suggestion
We plan to use your responses to support Nix Teams to better serve the community.
```